ui/jobs/jobs.js

```diff
@@ -1563,6 +1564,73 @@ class Bars {
     };
   }
 
+  // (In Progress) (Need Help)
+  setupSch() {
+    let gcd = kUnknownGCD;
+
+    let BioBox = this.addProcBox({
+      id: 'sch-procs-bio',
+      fgColor: 'sch-color-bio',
+      scale: gcd,
+      threshold: gcd + 1,
+    });
+
+    let AetherflowBox = this.addProcBox({
+      id: 'sch-procs-aetherflow',
+      fgColor: 'sch-color-aetherflow',
+      scale: gcd,
+      threshold: gcd * 3, // you need at most 3 gcd to cast all 3 stacks out
+    });
+
+    let AetherflowRescourceBox = this.addResourceBox({
+      classList: ['sch-color-aetherflow'],
+    });
+
+    let LucidDreamingBox = this.addProcBox({
+      id: 'sch-procs-luciddreaming',
+      fgColor: 'sch-color-lucid',
+      scale: gcd,
+      threshold: gcd + 1,
+    });
+
+    this.jobFuncs.push((jobDetail) => {
+      let aetherflow = jobDetail.aetherflowStacks;
+      AetherflowRescourceBox.innerText = aetherflow;
+      AetherflowBox.threshold = gcd * aetherflow;
```

Comment:
I don't think this box should have a threshold, as it's just a resource box?

Reply:
there is two box. AetherflowRescourceBox(maybe AFstackBox better) is a box that show your AF stacks.
AetherflowBox is a box that show AF cooldown. This cooldown box need a threshold.